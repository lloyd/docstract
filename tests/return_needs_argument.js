/**
 * a module with a function thats returns something unknown. error!
 */

/**
 * @function throweyFunc
 * @return
 */
