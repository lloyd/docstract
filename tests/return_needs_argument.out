{
  "args": [
    "@return tag requires an argument at line 5"
  ], 
  "exception_type": "<type 'exceptions.RuntimeError'>", 
  "stack": [
    [
      "<module>", 
      "got = ds.extractFromFile(os.path.join(testDir, test + \".js\"))"
    ], 
    [
      "extractFromFile", 
      "data = self.extract(contents)"
    ], 
    [
      "extract", 
      "self._analyzeBlock(block, context, firstBlock, data, lineStart, line)"
    ], 
    [
      "_analyzeBlock", 
      "self._consumeToks(tokens, parseData)"
    ], 
    [
      "_consumeToks", 
      "raise RuntimeError(\"%s tag requires an argument\" % cur)"
    ]
  ]
}
